### DISCUSSION TITLE
Migration to 3.0.0,  items not found in exports

### DISCUSSION THREAD

**Discussion by goto40** [eclipse-langium/langium#1405] [Q&A] [answered]
Thank you for your very nice product. It is very useful.
While migrating my toolchain I encountered some initial problems. Maybe I have overseen something...

Somehow I do not manage to import the follwoing items from `langium` or any of the added package export (https://www.typefox.io/blog/langium-release-3.0/)...

from `ast-utils.ts`:
* `streamAllContents` 
* `getDocument`
* `streamAst`
* `streamReferences`

from `cst-utils.ts`:
* `getNextNode`

**Comment by cdietrich** [comment, marked as answer]
You write `AstUtils.streamAllContents`. `CstUtils....` etc

**Reply by goto40** [reply]
Thank you for the super-quick reply. Works!

**Reply by cdietrich** [reply]
have concluded our migration already last week so i was prepared :)

**Comment by msujew** [comment]
FYI, those changes have been documented in the changelog as a breaking change:

https://github.com/eclipse-langium/langium/blob/main/packages/langium/CHANGELOG.md#breaking-changes
